readme.md

```diff
@@ -0,0 +1,165 @@
+# Custom Deployment
+
+## Learning Objectives
+
+- Configure a linux web server
+- Set up public key ssh authentication
+- Deploy a nodejs app
+- Configure a reverse proxy with nginx
+
+## Set up droplet
+
+Please use my referral code - well both get 10 bucks in credits!
+
+<http://jshawl.com/do>
+
+All the defaults are fine. Choose the cheapest droplet.
+
+All the way at the bottom, add your ssh key.
+
+You can get your ssh key by typing in
+
+```
+$ cat ~/.ssh/id_rsa.pub
+```
+
+This will allow you to log into the server without having to type your password.
+
+```
+$ ssh root@i.p.address.here
+```
+
+## Create a user and password
+
+```
+$ useradd -m theusernameyouwant -G sudo -s /bin/bash
+$ passwd theusernameyouchose
+```
+
+## `/etc/hosts`
+
+Remembering an IP address is difficult and time consuming. Let's fake a domain name
```

Comment:
maybe a little recap about how the IP address will be used in the scheme of things